package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "headquarters-node",
-  "version": "0.1.2",
+  "version": "0.2.0",
```

Comment:
There is a third way of looking at. It's not a totally new api, it's the same api under a different name. And the domain change introduced a bug in the api clients and se we patched the bug. On the standing point of this node client there isn't any new functionality. The way I see it, it doesn't matter where the api is. If it stopped working for the people using `0.1.2`, they could just upgrade to `0.1.3` knowing that nothing changed.